packages/eas-build-job/src/metadata.ts

```diff
@@ -2,6 +2,17 @@ import Joi from 'joi';
 
 import { Workflow } from './common';
 
+enum FingerprintSourceType {
+  'GCS' = 'GCS',
+  'PATH' = 'PATH',
+  'URL' = 'URL',
+}
+
+export type FingerprintSource =
+  | { type: FingerprintSourceType.GCS; bucketKey: string }
+  | { type: FingerprintSourceType.PATH; path: string }
+  | { type: FingerprintSourceType.URL; url: string };
```

Comment:
```suggestion
export enum FingerprintSourceType {
  'GCS' = 'GCS',
  'PATH' = 'PATH',
  'URL' = 'URL',
}

export type FingerprintSource =
  | { type: FingerprintSourceType.GCS; bucketKey: string }
  | { type: FingerprintSourceType.PATH; path: string }
  | { type: FingerprintSourceType.URL; url: string };
```

I believe that it would be good to export it in `index.ts` so it can be imported from `@expo/eas-build-job` in other places